rskj-core/src/main/java/co/rsk/peg/BridgeStorageConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package co.rsk.peg;
+
+import org.ethereum.config.BlockchainConfig;
+
+public class BridgeStorageConfiguration {
+    private Boolean isUnlimitedWhitelistEnabled;
+
+    public BridgeStorageConfiguration() {
+        this.isUnlimitedWhitelistEnabled = Boolean.FALSE;
+    }
+
+    public void setIsUnlimitedWhitelistEnabled(Boolean value) {
```

Comment:
This public `setIsUnlimitedWhitelistEnabled` method seems unnecessary:
* You could just call `bridgeStorageConfiguration.isUnlimitedWhitelistEnabled = config.isRfs170()` below
* You could also pass this value to the constructor

I prefer the latter